install/install.sh

```diff
@@ -121,7 +121,7 @@ read -e -r -p "GitHub Owner Name : " -i "CCExtractor" github_owner_name
 read -e -r -p "GitHub repository : " -i "ccextractor" github_repository
 read -e -r -p "Email Domain : " -i "${config_server_name}" email_domain
 read -e -r -p "Email API key (Generate one here https://www.mailgun.com/) : " -i "" email_api_key
-hmac_key=$(< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+hmac_key=$(head -80 /dev/urandom | LC_ALL=c tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
Can you explain what issues you ran into that caused you to change this?

Reply:
Running of this command ```< /dev/urandom tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1``` prints the hmac_key in the console, and this does not terminate until one manually cease it, due to this, the installation gets hung at this step (it did at least in my local computer).

Reply:
I can confirm the new command works on my system. The old one seems to be giving error.